var/spack/repos/builtin/packages/hicops/package.py

```diff
@@ -81,11 +81,9 @@ class Hicops(CMakePackage):
     depends_on("py-setuptools")
     depends_on("py-bottleneck")
     depends_on("py-pyparsing")
-    depends_on("py-subprocess32")
     depends_on("py-six")
     depends_on("py-setuptools-scm")
     depends_on("py-et-xmlfile")
-    depends_on("py-argparse")
```

Comment:
Argparse is part of the standard library in Python 2.7 and 3.2+, the `py-argparse` package is merely a backport of the standard library and is only needed when using Python 2.6 or 3.0–3.1. Hicops should have never listed this dependency to begin with since it uses Python 3.7+. 

P.S. All of the dependency types listed in hicops are incorrect. These should likely be `type=('build', 'run')`, the default is `type=('build', 'link')`. This means that when you `spack load hicops`, none of these dependencies are present in `PYTHONPATH`. But we can fix this in a separate PR.

Reply:
Thank you for clarifying @adamjstewart. We are planning to revamp the entire hicops package here once we merge the dev into production as there have been major changes in the dependencies! We can go ahead with these changes for now and can look at it later like you suggested,